wbooks/src/services/BookService.ts

```diff
@@ -1,14 +1,6 @@
 import { BOOKS_MOCK } from '@constants/mockBooks';
+import { FilteredBooks } from '@interfaces/Book';
 
 export const getBooks = () => Promise.resolve({ ok: true, data: BOOKS_MOCK, problem: '' });
 
-export const getFilteredBooks = (query: string) => {
-  if (!query) {
-    return Promise.resolve({ ok: true, data: { books: [], query }, problem: '' });
-  }
-
-  const filteredBooks = BOOKS_MOCK.filter(book =>
-    book.title.toLocaleLowerCase().includes(query.toLocaleLowerCase())
-  );
-  return Promise.resolve({ ok: true, data: { books: filteredBooks, query }, problem: '' });
-};
+export const getFilteredBooks = (data: FilteredBooks) => Promise.resolve({ ok: true, data, problem: '' });
```

Comment:
No, you must use the same `getBooks`,  this is wrong and unnecessary

Reply:
Done!